3648/3/db/seeds.rb

```diff
@@ -0,0 +1,12 @@
+Place.create(
+  place_name: 'McDonalds', location: 'St. 3/4',
+  description: 'Fast food', place_rating: 0
+)
+Place.create(
+  place_name: 'KFC', location: 'St. 5/6',
+  description: 'Better fast food', place_rating: 0
+)
+Place.create(
+  place_name: 'Burger King', location: 'St. 2/4',
+  description: 'Fast food', place_rating: 0
+)
```

Comment:
Use csv for seeds.
see https://github.com/rubizza-camp/summer-2019/pull/503/files#diff-f6f4886f1b4440176aada36024fd9501R14